api-reference/beta/api/directory_delete.md

```diff
@@ -0,0 +1,54 @@
+# Delete deletedItem
+
+Permanently delete an item in [deleted items](../resources/directory.md).
+
+### Prerequisites
+The following **scopes** are required to execute this API:
+* For users: *User.ReadWrite.All; Directory.Read.All; Directory.ReadWrite.All; Directory.AccessAsUser.All*
+* For groups: *Group.ReadWrite.All; Directory.Read.All; Directory.ReadWrite.All; Directory.AccessAsUser.All*
```

Comment:
I'm unsure if we need different permission scopes for users and groups.

This question is for all APIs.  You will see a similar breakout on each API.  For GET's it will be read only.  For DELETE and POST it will be read/write.